eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriter.java

```diff
@@ -56,10 +64,30 @@ public void open(ExecutionContext executionContext) throws ItemStreamException {
 
         Set<VCFHeaderLine> metaData = new HashSet<>();
         metaData.add(new VCFHeaderLine("reference", referenceAssembly));
-        metaData.add(new VCFInfoHeaderLine("VC", 1, VCFHeaderLineType.String,
+        metaData.add(new VCFInfoHeaderLine(VARIANT_CLASS_KEY, 1, VCFHeaderLineType.String,
                                            "Variant class according to the Sequence Ontology"));
-        metaData.add(new VCFInfoHeaderLine("SID", VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
+        metaData.add(new VCFInfoHeaderLine(STUDY_ID_KEY, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
                                            "Identifiers of studies that report a variant"));
+
+        metaData.add(new VCFInfoHeaderLine(VALIDATED_KEY, 0, VCFHeaderLineType.Flag,
+                                           "RS validated flag, present when the RS was validated by any method "
+                                           + "as indicated by the dbSNP validation status"));
```

Comment:
Oh crap, I forgot to drop the 'applies to other' because I also noticed it could be problematic :joy: But anyway, I'm fine with keeping them whole for consistency.

Reply:
But still, maybe we should try to use something like 'EVI' for evidence? One letter is just too short, if we ever decide to add anything that starts with the same...

Reply:
renamed to LOE, ASMM and ALMM